plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -175,19 +182,13 @@ export const Stepper = (stepperProps: StepperProps) => {
       setActiveStep(prevActiveStep => {
         const stepNum = prevActiveStep + 1;
         analytics.captureEvent('click', `Next Step (${stepNum})`);
+        analytics.captureEvent('click', `Next Step (${stepNum})`);
```

Comment:
Not sure we need this one anymore right?

Reply:
Good catch, missed it 👍